src/components/email/CustomEmail.tsx

```
@@ -0,0 +1,26 @@
+  title?: string;
+  preview?: string;
+};
+
+const CustomEmail = ({html, title, preview}: CustomEmailProps) => {
+  // Safety check - ensure html is a string
+  const safeHtml = typeof html === "string" ? html : String(html || "");
+
+  return (
+    <BaseLayout title={title || "Astro Starter"} preview={preview}>
+      {safeHtml.trim() ? (
+        <div dangerouslySetInnerHTML={{__html: safeHtml}} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)